example-adapter/pom.xml

```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.niis</groupId>
     <artifactId>example-adapter</artifactId>
-    <version>0.0.5-SNAPSHOT</version>
+    <version>0.0.6-SNAPSHOT</version>
```

Comment:
There's no need to increase the version number since the previous version (`0.0.5-SNAPSHOT`) was a snapshot version. We can continue to add new features to the snapshot, do a release (`0.0.5`) and update to `0.0.6-SNAPSHOT` after that.

Reply:
Changed back to 0.0.5-SNAPSHOT and updated on the servers as well.